src/main/java/io/codearte/jfairy/FairyModule.java

```diff
@@ -17,7 +13,7 @@
  * @author jkubrynski@gmail.com
  * @since 2013-11-15
  */
-class FairyModule extends AbstractModule {
+public abstract class FairyModule extends AbstractModule {
```

Comment:
If we're making this class abstract probably good idea is to provide some default implementation that will be used when no locale will be specified.